src/api/type-defs/inputs/CreateLabelsInput.ts

```diff
@@ -13,8 +13,9 @@ export default /* GraphQL */ `
     labeledDate: Date
     validation: ValidationInput
     userId: ID
-    imageId: ID
+    imageId: ID!
     objectId: ID
+    mlModel: String
```

Comment:
I believe that this was mistakenly omitted from the input. The code that expects this input examines the value for the `mlModel` property:

https://github.com/tnc-ca-geo/animl-api/blob/2c66fa1413d35fa6710486019548b64f2364efbc/src/api/db/models/Image.js#L715-L718